org.octave.Octave.json

```diff
@@ -0,0 +1,341 @@
+        }
+      ]
+    },
+    "shared-modules/glu/glu-9.0.0.json",
+    {
+      "name": "fltk",
+      "buildsystem": "cmake",
+      "config-opts": [
+        "-DOPTION_BUILD_SHARED_LIBS=ON",
+        "-DFLTK_LIBDIR:PATH=lib"
+      ],
+      "build-options": {
+        "cflags": "-fPIC",
+        "cxxflags": "-fPIC"
+      },
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://fltk.org/pub/fltk/1.3.4/fltk-1.3.4-1-source.tar.gz",
+          "sha256": "7fb2c8882433ce694e6900c94fda505e8f4ed3fa9c7e597007098a33b85c53f4"
+        }
+      ]
+    },
+    {
+      "name": "gnuplot",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/gnuplot/files/gnuplot/5.0.6/gnuplot-5.0.6.tar.gz",
+          "sha256": "5bbe4713e555c2e103b7d4ffd45fca69551fff09cf5c3f9cb17428aaacc9b460"
+        }
+      ]
+    },
+    {
+      "name": "ghostscript",
+      "config-opts": [
+        "--disable-cups"
+      ],
+      "build-options": {
+        "ldflags": "-L/app/lib"
+      },
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs921/ghostscript-9.21.tar.gz",
+          "sha256": "02bceadbc4dddeb6f2eec9c8b1623d945d355ca11b8b4df035332b217d58ce85"
+        }
+      ]
+    },
+    {
+      "name": "gl2ps",
+      "buildsystem": "cmake",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://geuz.org/gl2ps/src/gl2ps-1.4.0.tgz",
+          "sha256": "03cb5e6dfcd87183f3b9ba3b22f04cd155096af81e52988cc37d8d8efe6cf1e2"
+        }
+      ]
+    },
+    {
+      "name": "qhull",
+      "buildsystem": "cmake",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.qhull.org/download/qhull-2015-src-7.2.0.tgz",
+          "sha256": "78b010925c3b577adc3d58278787d7df08f7c8fb02c3490e375eab91bb58a436"
+        }
+      ]
+    },
+    {
+      "name": "epstool",
+      "no-autogen": true,
+      "make-install-args": [
+        "EPSTOOL_BASE=/app"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://pkgs.fedoraproject.org/repo/pkgs/epstool/epstool-3.08.tar.gz/465a57a598dbef411f4ecbfbd7d4c8d7/epstool-3.08.tar.gz",
+          "sha256": "f3f14b95146868ff3f93c8720d5539deef3b6531630a552165664c7ee3c2cfdd"
+        }
+      ]
+    },
+    {
+      "name": "hdf5",
+      "no-autogen": true,
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://support.hdfgroup.org/ftp/HDF5/current18/src/hdf5-1.8.19.tar.bz2",
+          "sha256": "59c03816105d57990329537ad1049ba22c2b8afe1890085f0c022b75f1727238"
+        }
+      ]
+    },
+    {
+      "name": "qrupdate",
+      "no-autogen": true,
+      "make-args": [
+        "solib",
+        "BLAS=-lopenblas",
+        "LAPACK="
+      ],
+      "make-install-args": [
+        "PREFIX=/app"
```

Comment:
Doesn't flatpak-builder do it by default?

Reply:
I'll check

Reply:
No, it doesn't.